assets/js/components/Energyflow/Energyflow.vue

```diff
@@ -5,29 +5,18 @@
 			@click="toggleDetails"
 		>
 			<Visualization
-				class="
-					col-12
-					offset-md-1
-					col-md-6
-					offset-lg-1
-					col-lg-8
-					offset-xl-1
-					col-xl-6
-					offset-xxl-1
-					col-xl-8
-					order-md-2
-				"
-				:showDetails="showDetails"
-				:gridImport="gridImport"
-				:selfConsumption="selfConsumption"
+				class="col-12 offset-md-1 col-md-6 offset-lg-1 col-lg-8 offset-xl-1 col-xl-6 offset-xxl-1 col-xl-8 order-md-2"
+				:show-details="showDetails"
+				:grid-import="gridImport"
+				:self-consumption="selfConsumption"
 				:loadpoints="loadpointsPower"
-				:pvExport="pvExport"
-				:batteryCharge="batteryCharge"
-				:batteryDischarge="batteryDischarge"
-				:pvProduction="pvProduction"
-				:homePower="homePower"
-				:batterySoC="batterySoC"
-				:valuesInKw="valuesInKw"
+				:pv-export="pvExport"
```

Comment:
Hilft es das überall zu ändern? Dann geht kein einfaches search-and-replace mehr wie beim letzten change :(

Reply:
Ja, da muss man dann beide Schreibweisen searchen and replacen. Das ist die Vue recommended Schreibweise. Ich bin da her offen. Wir können auch camelcase nehmen. Wichtig ist mir nur, dass wir nicht mischen.